docs/src/development/local-collaboration.md

```diff
@@ -33,10 +33,12 @@ To use a different set of admin users, you can create your own version of that j
 
 ## Testing collaborative features locally
 
-In one terminal, run Zed's collaboration server and the `livekit` dev server:
+Ensure that Postgres is configured and running, then run Zed's collaboration server and the `livekit` dev server:
 
 ```sh
 foreman start
+# OR
+cargo run -p collab -- serve all
```

Comment:
I think this is not enough to test all collaboration features locally, as for voice & screenshare we need `livekit`.

More proper command would be `docker compose up` which will take care of that + Postgres + the rest via https://github.com/zed-industries/zed/blob/main/compose.yml